src/partials/template.html

```diff
@@ -0,0 +1,20 @@
+<head>
+    <title></title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <style type="text/css">
+      html,
+      body {
+        height: 100%;
+        margin: 0;
+      }
+  
+      #full {
+        min-height: 100%;
+      }
+  
+    </style>
+  </head>
+  
+  <body>
+    <div id="full">No map to display</div>
```

Comment:
What's the meaning of the "full" id? Not an explicit name

Reply:
It is the html div container for the map. I rename it to "container"

Reply:
done